sematic/ee/plugins/external_resource/ray/load_testing/__main__.py

```diff
@@ -0,0 +1,208 @@
+"""
+Entry point for the testing pipeline.
+"""
+# Standard Library
+import argparse
+import logging
+import os
+from typing import Optional
+
+# Third-party
+import debugpy  # type: ignore
+
+# Sematic
+from sematic import CloudResolver, LocalResolver, Resolver, SilentResolver
+from sematic.ee.plugins.external_resource.ray.load_testing.pipeline import (
+    CollatzConfig,
+    MnistConfig,
+    load_test_ray,
+)
+
+logger = logging.getLogger(__name__)
+
+BAZEL_COMMAND = (
+    "bazel run //sematic/ee/plugins/external_resource/ray/load_testing:__main__ --"
+)
+DESCRIPTION = (
+    "This pipeline is used to perform load testing on the Ray integration. "
+    "There are two kinds of load test: Collatz and MNIST. Collatz load testing "
+    "uses a large number of independent tasks with varying (but small) compute "
+    "and memory usage needs. It can be configured to begin OOM'ing tasks for "
+    "some workloads. MNIST load testing performs GPU training of an MNIST model "
+    "with a variety of learning rates."
+)
+CLOUD_HELP = (
+    "Whether to run the resolution in the cloud, or locally. Defaults to False. "
+    "Only one of --silent or --cloud are allowed."
+)
+SILENT_HELP = (
+    "Whether to run the resolution using the SilentResolver. Defaults to False. "
+    "Only one of --silent or --cloud are allowed."
```

Comment:
This is not enforced in the args processing. Please handle.

Reply:
done